packages/govern-contract-utils/contracts/bitmaps/BitmapLib.sol

```diff
@@ -0,0 +1,19 @@
+/*
+ * SPDX-License-Identifier:    GPL-3.0
+ */
+
+pragma solidity 0.6.8;
+
+library BitmapLib {
```

Comment:
Pretty nifty! Should we add some docs on how this works? I don't expect most frontend developers to be very familiar with bitwise operations.

Reply:
That'd be great. I think we should have an abstraction similar to how we do in the tests (an array of bools on whether things can fail which can be converted, more efficiently, to the bitmap)